lib/src/main/java/io/ably/lib/http/Http.java

```diff
@@ -189,7 +189,9 @@ private String getAuthorizationHeader(boolean renew) throws AblyException {
 		if(auth.getAuthMethod() == AuthMethod.basic) {
 			authHeader = "Basic " + Base64Coder.encodeString(auth.getBasicCredentials());
 		} else {
-			auth.authorise(null, null, renew);
+			Auth.AuthOptions options = new Auth.AuthOptions();
```

Comment:
It doesn't seem like there is a difference between using,
  - null argument, and letting system use default options
  - using an empty authoptions and letting system merge it with default options

i.e.,

https://github.com/ably/ably-java/blob/5987ee1292a72dc2ba8415b2bb0e705aac139bd6/lib/src/main/java/io/ably/lib/rest/Auth.java#L334

https://github.com/ably/ably-java/blob/5987ee1292a72dc2ba8415b2bb0e705aac139bd6/lib/src/main/java/io/ably/lib/rest/Auth.java#L455